src/Nethermind/Nethermind.Core2.Abstractions/INetworkPeering.cs

```diff
@@ -15,16 +15,22 @@
 //  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
 
 using System.Threading.Tasks;
-using Nethermind.Core2.Containers;
 using Nethermind.Core2.Crypto;
+using Nethermind.Core2.P2p;
 using Nethermind.Core2.Types;
 
 namespace Nethermind.Core2
 {
     public interface INetworkPeering
```

Comment:
not sure if this will be the same - in Eth1 we have a EthSyncPeerPool which is capable of serving peers to sync with based on some criteria and it can do that to support massive concurrency of sync requests,

Reply:
Hmm. Not even sure if I can control that through Mothra. The API doesn't have any control over peers, so it might handle that internally. I'm probably going to have to run up a multiple (more than 3) test net to check... once I get sync working.